apps/liteprotocoltester/run_service_node.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+echo "I am a service node"
+IP=$(ip a | grep "inet " | grep -Fv 127.0.0.1 | sed 's/.*inet \([^/]*\).*/\1/')
+
+echo "Service node IP: ${IP}"
+
+exec /usr/bin/wakunode\
+      --relay=true\
+      --filter=true\
+      --lightpush=true\
+      --store=false\
+      --rest=true\
+      --rest-admin=true\
+      --rest-private=true\
+      --rest-address=0.0.0.0\
+      --rest-allow-origin="waku-org.github.io"\
+      --rest-allow-origin="*"\
```

Comment:
shouldn't it be enough with the `*`?

Reply:
Yeah, probably would do! I will remove the first one.